plat/common/arm/psci_arm64.S

```diff
@@ -38,6 +38,7 @@
  * int32_t smcc_psci_hvc_call(uint32_t, uint64_t, uint64_t, uint64_t);
  */
 ENTRY(smcc_psci_hvc_call)
+	bti j
```

Comment:
These instructions should be conditional to CONFIG_ARM64_FEAT_BTI. Otherwise will break the build for older compilers (i.e. gcc < 9).

Reply:
Good point. Actually this commit can be removed if PSCI calls are migrated to use the SMCCC API. @razvanvirtan, the BTI PR is scheduled for this release. Are you planning to include the PSCI change on this release too? If not I can push a update on this PR with @rene's suggestion.

Reply:
@michpappas I have created the PSCI change PR: https://github.com/unikraft/unikraft/pull/428
From what I see, this PR (the BTI one) is planed for the next release (0.9), so I think the PSCI one can be included in that release too.

Reply:
@razvanvirtan awesome, thanks a lot 🚀 I'll look at it this weekend. @razvand could we schedule this one for 0.9 too?